src/main/resources/openapi/event.yaml

```diff
@@ -436,5 +512,14 @@ components:
         enum:
           - "Data on the event could not be reconciled with Darts records"
           - "No event handler found in database"
-        x-enum-varnames: [ EVENT_DATA_NOT_FOUND, EVENT_HANDLER_NOT_FOUND_IN_DB]
+          - "Event mapping already exists in database"
+          - "Handler name does not exist"
+        x-enum-varnames: [ EVENT_DATA_NOT_FOUND, EVENT_HANDLER_NOT_FOUND_IN_DB, DUPLICATE_EVENT_MAPPING, INVALID_HANDLER_MAPPING_NAME]
+
+    PostEventMappingErrorCode:
+      type: string
+      enum:
+        - "EVENT_102"
+        - "EVENT_103"
+      x-enum-varnames: [ DUPLICATE_EVENT_MAPPING, INVALID_HANDLER_MAPPING_NAME ]
```

Comment:
Generally we define all the error values in a single enum, which makes it easier to reuse them across different scenarios. Suggest moving these into `EventErrorCode`.